### PR TITLE
fix(suite-native): display fiat value skeleton while fiat rates are loading

### LINKED ISSUE #13299: Loading fiat value should show skeleton instead of 0 value
**Describe the bug**
- fiat values of portfolio are not cached
- when you start app when offline you see all fiat values with zero value
- before fiat values are fetched from servers after you see start app all values are zero too for quite a long time
- same results for latest dev version, 24.7.1 version and two weeks old dev version

**Steps to reproduce:**
1. Import multiple accounts
2. Kill app
3. Start app with phone in flight mode or observe app before values are fetched from servers

**Info:**
 - Suite version: Release notes for 24.6.0 (350)
Last commit hash: 8a20d9cf4567a8077dd95d88212f9bd2eb66f89e

**Screenshots:**

telegram-cloud-photo-size-4-5801191960428528094-y

https://github.com/trezor/trezor-suite/assets/31506317/f965e330-d720-4c4c-add4-57c88a00244a

### PR DESCRIPTION
**Pull request by yanascz**
Make sure we handle fiat values as `null` until fiat rates are loaded.

## Related Issue

Resolve #13299

## Screenshots:

https://github.com/user-attachments/assets/c3f89c0f-47ff-4a42-9b79-1d34884bae0f

### CODE REVIEW

File: suite-native/assets/src/assetsSelectors.ts [hunk 1]
```diff
@@ -80,22 +77,19 @@ export const selectDeviceAssetsWithBalances = (state: AssetsRootState & DeviceRo
...
-                return prev;
-            }, 0)
-            .toFixed();
-
-        const cryptoValue = sumCryptoBalance(networkAccounts.map(account => account.cryptoValue));
+        const assetBalance = networkAccounts.reduce(
+            (sum, { cryptoValue }) => sum.plus(cryptoValue),
+            new BigNumber(0),
+        );
+        const fiatBalance = networkAccounts.reduce(
```
Comment (Nodonisko) [inline suite-native/assets/src/assetsSelectors.ts:84]: I am quite concern about performance here, this selector is already on my list for being slow and BigNumber is one of biggest perf bottlenecks.

File: suite-native/formatters/src/components/FiatAmountFormatter.tsx [hunk 2]
```diff
@@ -21,9 +21,12 @@ export const FiatAmountFormatter = ({
 }: FiatAmountFormatterProps) => {
     const { FiatAmountFormatter: formatter } = useFormatters();
 
-    const isTestnetValue = !!network && isTestnet(network);
```
Comment (Nodonisko) [inline suite-native/formatters/src/components/FiatAmountFormatter.tsx:24]: Can you please keep it as variable?

File: suite-native/assets/src/assetsSelectors.ts (hunk 1 above)
Comment (yanascz) [reply suite-native/assets/src/assetsSelectors.ts:84]: `BigNumber` was already used to add crypto amounts and is also used elsewhere when dealing with fiat amounts. I believe it makes sense considering the rounding errors with `number`.

What other solution did you have in mind?

File: suite-native/formatters/src/components/FiatAmountFormatter.tsx (hunk 2 above)
Comment (yanascz) [reply suite-native/formatters/src/components/FiatAmountFormatter.tsx:24]: What added value does it have?

File: suite-native/assets/src/assetsSelectors.ts (hunk 1 above)
Comment (yanascz) [reply suite-native/assets/src/assetsSelectors.ts:84]: Changed back to `Number` based on our discussion in person.

**Review by Nodonisko — APPROVED** [review]

File: suite-native/formatters/src/components/FiatAmountFormatter.tsx (hunk 2 above)
Comment (Nodonisko) [reply suite-native/formatters/src/components/FiatAmountFormatter.tsx:24]: It's easier to understand what `if` condition is for. Here is relatively simple so it's probably fine dropping it but in many cases it helps a lot.

File: suite-native/module-accounts-management/src/components/AccountDetailGraphHeader.tsx [hunk 3]
```diff
@@ -66,26 +47,17 @@ const CryptoBalance = ({
...
         selectAccountByKey(state, accountKey),
     );
-
     const tokenSymbol = useSelector((state: AccountsRootState) =>
         selectEthereumAccountTokenSymbol(state, accountKey, tokenAddress),
     );
+    const setPoint = useSetAtom(selectedPointAtom);
 
-    const firstGraphPoint = useAtomValue(referencePointAtom);
+    // Reset selected point on unmount so that it doesn't display on device change
```
Comment (matejkriz) [inline suite-native/module-accounts-management/src/components/AccountDetailGraphHeader.tsx:59]: 👏😍

**Pull request merged**